literature.bib

```diff
@@ -10522,6 +10866,35 @@ @article{BSRC10
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2009 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{FHS:SPLC09,
+	title = {{Inferring Information From Feature Diagrams to Product Line Economic Models}},
+	author = {David Fern{\'{a}}ndez{-}Amor{\'{o}}s and Ruben Heradio Gil and
+               Jos{\'{e}} Antonio Cerrada Somolinos},
+	booktitle = SPLC,
+	pages = {41--50},
+	year = {2009},
+	publisher = Springer,
+	address = BerlinHeidelberg
+}
+
+@inproceedings{MBC:OOPSLA09,
+  title = {{SPLOT: Software Product Lines Online Tools}},
+  author = {Mendon\c{c}a, Marcilio and Branco, Moises and Cowan, Donald},
+  booktitle = OOPSLA,
+  pages = {761--762},
+  publisher = ACM,
+  address = NY,
+  year = 2009
+}
+
+@misc{Tiu09,
+	title = {{Introduction to Logic}},
+	howpublished = {\url{http://ssll.rsise.anu.edu.au/files/slides/tiu.pdf}},
+	author = {Tiu, Alwen},
+	year = 2009,
+	notes = {Slides, Accessed: 2020-12-22}
+}
+
```

Comment:
There is actually a dedicated misc section at the bottom of this file, where misc entries are gathered, independent of the year. How to proceed @tthuem ?

Reply:
Moved.